frontend/src/component/FeedItem/Preview.jsx

```diff
@@ -236,28 +241,56 @@ export class Preview extends React.Component {
 
   getPreviewComponent () {
     const { props, state } = this
+    let component = null
     if (state.previewLoading) {
-      return <>{props.t('Preview loading...')}</>
+      component = <>{props.t('Preview loading...')}</>
```

Comment:
```suggestion
      component = props.t('Preview loading...')
```
maybe rename `component` to `preview` if you really don't like storing a string in a variable named `component`, I personally don't mind either way.

Reply:
In fact this was already in a react fragment so I didn't change it.

Reply:
I'd remove it since it does useless calls to React's virtual DOM APIs, but if you prefer to keep it as is I am ok.